### DISCUSSION TITLE
Welcome to signals Discussions!

### DISCUSSION THREAD

**Discussion by scarplus** [scarplus/signals#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BLTCPT** [comment]
Great work here. I am wondering how you looked in Coles Win Above Replacement Portfolio (CWARP) ?

**Reply by scarplus** [reply]
I haven't. could you share link ?

**Reply by BLTCPT** [reply]
https://www.artemiscm.com/cwarp for Money Ball Theory https://docsend.com/view/teaqrcewe7ht423x
